src/applications/hca/components/FormFields/VaMedicalCenter.jsx

```diff
@@ -109,23 +172,52 @@ const VaMedicalCenter = props => {
   }
 
   return !error ? (
```

Comment:
I know this was already like this, but I think this would read clearer if we replaced this `!error` ternary with a `if(error) {  <ServerErrorAlert /> }` and then after that just return our `VaSelect` fields. That way you can follow the different logic paths down in the if statements.

Reply:
Agreed, a Guard Clause pattern would be tidier here to get the error case out of the way.